applicationconfig/client/README.md

```diff
@@ -0,0 +1,84 @@
+# App Configuration client library for Java
+Azure App Configuration allows developers to store, retrieve and manage access to application settings all in one place. It is easy to set up and simple to use from any application. It gives developers the ability to modify an application's behavior on demand without having to redeploy the application.
```

Comment:
I am curious to know where this description came from... I will stay away from saying things like `easy` 
=> BTW in .NET we said `...that allows to easily store and manage ...` I got feedback about that word too, so I need to change it.
I was thinking on using the description the service uses... like 

*Azure App Configuration provides a service to centrally manage application settings. Modern programs, especially programs running in a cloud, generally have many components that are distributed in nature. Spreading configuration settings across these components can lead to hard-to-troubleshoot errors during an application deployment. Use App Configuration to store all the settings for your application and secure their accesses in one place.*

What do you think?

Reply:
It came from the [Overview](https://docs.microsoft.com/en-us/azure/azure-app-configuration/overview) section in the App Configuration docs.

I don't mind the wording change especially if it keeps us consistent across languages.

Reply:
hmm I don't see it there, but either way, I think you should include more info about the service like with what I suggested. 
As it currently stands it is a little too generic? or simple?